src/main/java/io/milvus/param/R.java

```diff
@@ -60,17 +62,17 @@ public void setData(T data) {
         return r;
     }
 
-    public static <T> R<T> failed(ErrorCode errorCode) {
+    public static <T> R<T> failed(ErrorCode errorCode, String msg) {
```

Comment:
Why do we directly call failed(ErrorCode errorCode, Exception e)?

Reply:
No need to always construct an Exception object, easy to call.